content/desktop-environment.md

```diff
@@ -37,14 +45,16 @@ For Ubuntu, install the `ubuntu-desktop` package instead:
 sudo apt install ubuntu-desktop
 ```
 
-The desktop environment acts as the top graphical layer of the OS. The desktop environment is launched by a display manager; Pop!\_OS and Ubuntu both use GDM (GNOME Display Manager) by default.
-
-If multiple desktop environments are installed, GDM will display a gear icon, which will allow you to select the desktop environment you want to launch. You will need to either reboot or restart your display manager using `sudo systemctl restart gdm` before a newly-installed desktop environment will show up in the list of options.
+In some instances, icons and display settings may not automatically revert to their original values. To resolve this, install the GNOME Tweaks utility, navigate to the `Appearance` tab, then restore `Themes` settings to the `Pop` options. GNOME Tweaks can be installed from the Pop!\_Shop, or with this command:
 
-![Login2](/images/desktop-environment/Login2.png)
+```bash
+sudo apt install gnome-tweaks
+```
 
 ## Different Desktop Environments
 
+The desktop environment acts as the top graphical layer of the OS. The desktop environment is launched by a display manager; Pop!\_OS and Ubuntu both use GDM (GNOME Display Manager) by default.
```

Comment:
I suggest we merge this PR and then move this section further up on the article.